Makefile

```diff
@@ -158,19 +165,18 @@ format: ; $(info $(M) running format )
 
 .PHONY: help
 help: ## Shows this help message
-	$(Q) echo 'usage: make [target] ...'
-	$(Q) echo
-	$(Q) echo 'targets : '
-	$(Q) echo
-	$(Q) fgrep -h "##" $(MAKEFILE_LIST) | fgrep -v fgrep | sed -e 's/\\$$//' | sed -e 's/##//'| column -s: -t
+	$(Q) awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n\nTargets:\n"} /^[a-zA-Z_-]+:.*?##/ { printf "  \033[36m%-15s\033[0m\t %s\n", $$1, $$2 }' $(MAKEFILE_LIST)
 
-### Dependencies
+# Dependencies
 
 $(GOTEST_BIN): ; $(info $(M) getting gotest )
-	$(Q) GO111MODULE=off $(GOGET) -u github.com/rakyll/gotest
+	$(Q) $(GOBUILD) -o $@ github.com/rakyll/gotest
 
 $(EMBEDMD_BIN): ; $(info $(M) getting embedmd )
-	$(Q) GO111MODULE=off $(GOGET) -u github.com/campoy/embedmd
+	$(Q) $(GOBUILD) -o $@ github.com/campoy/embedmd
+
+$(LICHE_BIN): ; $(info $(M) getting liche )
+	$(Q) $(GOBUILD) -o $@ github.com/raviqqe/liche
 
 $(GOLANGCI_LINT_BIN): ; $(info $(M) getting golangci-lint )
 	$(Q) curl -sfL https://raw.githubusercontent.com/golangci/golangci-lint/$(GOLANGCI_LINT_VERSION)/install.sh \
```

Comment:
Maybe you should also move `golangci-lint` and `goreleaser` to `tools.go`

Reply:
`golangci-lint` should be moved for sake of consistency. However, I don't think, I'll move `goreleaser`. I even think I should remove it from `Makefile`.